core/src/main/clojure/xtdb/types.clj

```diff
@@ -717,55 +721,76 @@
   ^bytes [s]
   (.getBytes (str s) StandardCharsets/UTF_8))
 
+(def iso-local-date-time-formatter-with-space
```

Comment:
now that pgwire's in core I'd be tempted to shift the pg-specific stuff out to xtdb.pgwire.types or similar?